src/platformIcon.tsx

```diff
@@ -243,10 +231,30 @@ export const PLATFORM_TO_ICON = {
   // Please add them where they belong alphabetically
 } as const;
 
+
+// DO NOT EXPORT THIS MAP. 
+// It's in place to ensure compatibility with existing code that uses the "node" prefix.
+const NODE_TO_JS_MAP = {
+  "node-awslambda": "javascript-awslambda",
+  "node-azurefunctions": "javascript-azurefunctions",
+  "node-cloudflare-pages": "javascript-cloudflare-pages",
+  "node-cloudflare-workers": "javascript-cloudflare-workers",
+  "node-connect": "javascript-connect",
+  "node-express": "javascript-express",
+  "node-fastify": "javascript-fastify",
+  "node-gcpfunctions": "javascript-gcpfunctions",
+  "node-hapi": "javascript-hapi",
+  "node-koa": "javascript-koa",
+  "node-nestjs": "javascript-nestjs",
+  "node-serverlesscloud": "javascript-serverlesscloud",
+}
+
 function normalizePlatform(platform: string): string {
   // sentry uses format python-django, but docs uses python.django
   // this function normalizes that
-  return platform.replace(".", "-");
+  const dashedPlatform =  platform.replace(".", "-");
+
+  return NODE_TO_JS_MAP[dashedPlatform as keyof typeof NODE_TO_JS_MAP] ?? dashedPlatform
```

Comment:
I would have just done a simple search and replace. I don't see a reason to limit is to the existing "node platforms". Otherwise, we again have the situation that contributors add platforms with both javascript and node prefix.
```suggestion
  // Allow `node` as an alias for `javascript` to ensure backwards compatibility
  return dashedPlatform.replace(/^node-/, 'javascript-')
```

Reply:
this works too ^^